components/Tabs/Tabs.js

```diff
@@ -2,50 +2,45 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element= element;    
     // Get the custom data attribute on the Link
-    // this.data;
+     this.data = this.element.dataset;
     
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+     this.itemElement = this.data.tab;
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+     this.tabItem = new TabItem(document.querySelector('.tabs-item[data-tab="'+this.itemElement+'"]'));
     
```

Comment:
Line 12 can be made more readable by saving the result of your use of `.querySelector` in a separate variable (which was also what was intended by the earlier commented instructions).